src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -44,121 +61,319 @@ export function formatDate(date) {
         <div>
-          The Board of Veterans Appeals has made a decision on some of your appeals.
-          You will receive your decision letter in the mail in {businessDays} business days. Here is an overview
-          of the decision:
+          <p>
+            The Board of Veterans’ Appeals made a decision on your appeal. Here is an overview of
+            the decision:
+          </p>
+          <div className="decision-items">
+            {allowedBlock}
+            {deniedBlock}
+            {remandBlock}
+          </div>
+          <p>
+            If this decision changes your disability rating or your eligibility for VA benefits,
+            you should expect this adjustment to be made in 1 to 2 months.
+          </p>
         </div>
-        <br/>
-        <div className="decision-items">
-          <h5 className="allowed-items">Allowed</h5>
-          <ul>{allowedIssues}</ul>
-          <h5 className="denied-items">Denied</h5>
-          <ul>{deniedIssues}</ul>
-          <h5 className="remand-items">Remand</h5>
-          <ul>{remandIssues}</ul>
+      );
+      break;
+    }
+    case STATUS_TYPES.bvaDecision: {
+      const { decisionIssues } = details;
+      const allowedIssues = decisionIssues
+        .filter((issue) => (issue.disposition === 'allowed'))
+        .map((issue, i) => (<li key={`allowed-${i}`}>{issue.description}</li>));
+      const deniedIssues = decisionIssues
+        .filter((issue) => (issue.disposition === 'denied'))
+        .map((issue, i) => (<li key={`denied-${i}`}>{issue.description}</li>));
+
+      const pluralize = {
+        allowed: (allowedIssues.length > 1) ? 'these issues' : 'this issue',
+        denied: (deniedIssues.length > 1) ? 'issues' : 'issue',
+      };
+
+      let allowedBlock = null;
+      let deniedBlock = null;
+
+      if (allowedIssues.length) {
+        allowedBlock = (
+          <div>
+            <h5 className="allowed-items">Allowed</h5>
+            <p>The judge overrules the original decision and finds {pluralize.allowed} in your favor</p>
+            <ul>{allowedIssues}</ul>
+          </div>
+        );
+      }
+      if (deniedIssues.length) {
+        deniedBlock = (
+          <div>
+            <h5 className="denied-items">Denied</h5>
+            <p>The judge upholds the original decision for the following {pluralize.denied}</p>
+            <ul>{deniedIssues}</ul>
+          </div>
+        );
+      }
+
+      contents.title = 'The Board made a decision on your appeal';
+      contents.description = (
+        <div>
+          <p>
+            The Board of Veterans’ Appeals made a decision on your appeal. Here is an overview of
+            the decision:
+          </p>
+          <div className="decision-items">
+            {allowedBlock}
+            {deniedBlock}
+          </div>
+          <div>
+            For issues that are allowed, you will receive compensation. For more information, please
+            contact your VSO or representative.
```

Comment:
This should be:

> If this decision changes your disability rating or eligibility for VA benefits, you should expect this adjustment to be made in 1 to 2 months.

Reply:
VSO should be spelled out. Use Veteran Service Organization.

Reply:
Gah, nice catch!